src/components/MoistureCard.vue

```diff
@@ -16,12 +16,13 @@
 </template>
 
 <script>
-import Progress from "easy-circular-progress";
+// Weg gedaan door matthias
```

Comment:
Weet niet of dit beter is dan wat we nu hebben
![Screenshot 2021-11-07 at 13 31 36](https://user-images.githubusercontent.com/25200049/140644992-9d617477-cba2-4d0b-8aa6-5089e383b1ac.png)
hier kan ik niet zien wat de waarde van de sensor is

dit is hoe het er nu uit ziet
![Screenshot 2021-11-07 at 13 32 38](https://user-images.githubusercontent.com/25200049/140645042-42f4c2aa-e192-4112-a992-6c71d804874e.png)

de blauwe progess bar is wel mooi maar het blijft op 100% staan en de waarde wordt ook niet weergegeven dus misschien die 2 dingen aanpassen.